lib/drivers/bayer/bayerContourNext.js

```diff
@@ -517,6 +529,132 @@ module.exports = function (config) {
     return dataToPost;
   };
 
+  var setDateTime = function(serverTime, cb) {
+    var ACK_ERROR = 'Expected ACK during connect:';
+
+    async.series({
```

Comment:
Is it worth refactoring this to use `async/await`? Or is that too much for now?

Reply:
Converting this driver to ES6 is maybe a bit too big of a push for now :smile: